numalogic/models/autoencoder/variants/conv.py

```diff
@@ -233,15 +232,15 @@ def __init__(
                 enc_kernel_sizes
```

Comment:
I think we should not use `assert` in production code.

https://snyk.io/blog/the-dangers-of-assert-in-python/

Reply:
Very true. Will fix very soon. Tracking here: https://github.com/numaproj/numalogic/issues/169